app/lib/delegates/js_base.rb

```diff
@@ -0,0 +1,29 @@
+class Delegates::JSBase
+  include Capybara::DSL
+
+  attr_reader :message
+
+  def initialize(message)
+    @message = message
+    configure_capybara
+  end
+
+  def deliver!
+    raise NotImplementedError
+  end
+
+  protected
+
+  def host
+    raise NotImplementedError
+  end
+
+  def configure_capybara
+    Capybara.configure do |config|
```

Comment:
Is there a way to instead do: `Capybara.new(...options...)` instead of reinvoking the global configurator each time?